### PR TITLE
[16543] Fix muting channel from inside and outside chat page

### LINKED ISSUE #16543: "Mute channel" doesn't work in the channel context menu
# Bug Report

## Problem 

You can only mute a channel from the long-press menu on the community screen (channel list), but if you open the channel, open the context menu, and try to mute it from there, it won't work.

https://github.com/status-im/status-mobile/assets/67952253/c90d9c1a-5dd8-4b41-adf5-180dfb82a810

#### Expected behavior
Can mute the channel from both the long-press menu and the menu inside the channel

#### Actual behavior
`Mute channel` inside the channel doesn't work

### Reproduction

1. Open community
2. Open channel
3. Tap `...` at the top right 
4. Mute the channel
5. Check if it worked 

### Additional Information

- Status version: nightly 10/07
- Operating System: Android, iOS

[comment]: # (Please, add logs/notes if necessary)

### PR DESCRIPTION
**Pull request by ibrkhalil**
fixes #16543 

### Summary
Fixes an inconsistency in how props are configured in the chat actions bottom sheet.
 
status: ready

### CODE REVIEW

File: src/status_im2/contexts/communities/overview/view.cljs [hunk 1]
```diff
@@ -51,7 +51,7 @@
 (defn- channel-chat-item
   [community-id community-color {chat-id :id muted? :muted? :as chat}]
   (let [sheet-content      [actions/chat-actions
-                            (assoc chat :chat-type constants/community-chat-type)
+                            (assoc chat :chat-type constants/community-chat-type :chat-id (str community-id chat-id))
```
Comment (Parveshdhull) [inline src/status_im2/contexts/communities/overview/view.cljs]: Hi @ibrkhalil, Thank you very much for working on the issue.
Please can you clarify, how this fixes the issue?

Comment (Parveshdhull) [reply src/status_im2/contexts/communities/overview/view.cljs]: Is `id` is nil in above cases?

Comment (ibrkhalil) [reply src/status_im2/contexts/communities/overview/view.cljs]: Yes

Comment (Parveshdhull) [reply src/status_im2/contexts/communities/overview/view.cljs]: And why `chat-id` crated by `community-id` and `chat-id` itself?

Comment (ibrkhalil) [reply src/status_im2/contexts/communities/overview/view.cljs]: That's how we reference a community channel chat, We concatenate the community-id with the id of that chat.

Comment (Parveshdhull) [reply src/status_im2/contexts/communities/overview/view.cljs]: Looks like we are modifying wrong file here, this bottom-sheet is dispatched by event in `status-im2.contexts.chat.messages.navigation.view` ([ref](https://github.com/status-im/status-mobile/blob/728d9d01c500ab1d489f93f5e006e9d83b881a5b/src/status_im2/contexts/chat/messages/navigation/view.cljs#L106))
And, passed chat in that event already has a chat-id, so this change is not required

Comment (Parveshdhull) [reply src/status_im2/contexts/communities/overview/view.cljs]: Ok, this change will make sure that, new changes don't break in communities overview screen.

**Review by Parveshdhull — APPROVED** [review]

File: src/status_im2/contexts/communities/overview/view.cljs (hunk 1 above)
Comment (Parveshdhull) [reply src/status_im2/contexts/communities/overview/view.cljs]: Please can you remove, renaming of `id` to `chat-id`.
`{chat-id :id muted? :muted? :as chat}`
Looks little confusing, that :chat-id is created from :chat-id itself.

**Review by siddarthkay — APPROVED** [review]

File: src/status_im2/contexts/communities/overview/view.cljs (hunk 1 above)
Comment (ibrkhalil) [reply src/status_im2/contexts/communities/overview/view.cljs]: Done, Thanks for the quick review Parvesh!

**Comment by qoqobolo** [comment]
Thanks for the quick fix @ibrkhalil!
PR can be merged.

Could you also rebase https://github.com/status-im/status-mobile/pull/16528 once this one is merged?

**Comment by ibrkhalil** [comment]
> Thanks for the quick fix @ibrkhalil!
> 
> PR can be merged.
> 
> 
> 
> Could you also rebase https://github.com/status-im/status-mobile/pull/16528 once this one is merged?
> 
> 

Okay

**Pull request merged**